src/status_im/navigation/screens.cljs

```diff
@@ -489,6 +490,11 @@
                  {:modalPresentationStyle :overCurrentContext})
      :component wallet-scan-address/view}
 
+    {:name      :screen/wallet-swap.select-asset-to-pay
```

Comment:
nit: `swap` is just another part of wallet. should the screen name not reflect this the same for all wallet pieces??
e.g
`:screen/wallet.swap-select-asset-to-pay`,
i.e let's not combine team structure with code structure.

but your idea here could also be an improvement on the organisation of screen keys if we adopt it more across the other screens 👍

Reply:
Yes, that's true, I will rename the screen name. We can refactor screen names later if we think other format is better 👍